html-css/week2/style.css

```diff
@@ -0,0 +1,190 @@
+:root {
+  --white: #ffffff;
+  --lith-gray: #eae8e8;
```

Comment:
what is `lith-gray`? Is it some special name for gray color? 🤔

Reply:
It is because I have 3 different gray colors. Maybe you can show another way for naming?
![image](https://user-images.githubusercontent.com/73549460/184890092-97be3a91-c2a8-47fa-9621-4e829b6205bc.png)

Reply:
OH! It is the word with mistake))) "light"